src/pages/Sign/SignIn/InputBox.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const InputBox = ({ data, onChange }) => {
+  const { name, placeholder, type } = data;
+  return (
+    <InputValue
+      name={name}
+      placeholder={placeholder}
+      type={type}
+      onChange={onChange}
+      autoComplete={type}
+    />
+  );
+};
+
+const InputValue = styled.input`
+  height: 60px;
+  width: 100%;
+  padding-left: 20px;
+  border: 1px solid #d7d7d7;
+  border-radius: 5px;
+  font-size: 20px;
+  margin: 10px 0px 10px 0px;
```

Comment:
아래와 같이 변경해주시면 좋겠습니다 :) 
그리고 css 비슷한 속성끼리 묶어주세요 :) !!
```suggestion
  margin: 10px 0px;
```